frontend/src/Components/CompanyDetails/CompanyDetails.jsx

```diff
@@ -1,68 +1,63 @@
 import Events from "./Events"
 import Orders from "./Orders"
-import { Table, TableBody, TableCell, TableContainer, TableRow } from "@material-ui/core"
+import ContactPersons from "./ContactPersons"
+import Notes from "./Notes"
+import CompanyInformations from "./CompanyInformations"
+
 import Button from "@material-ui/core/Button"
-import DeleteDialog from "./DeleteDialog"
-import React from "react"
 import Typography from "@material-ui/core/Typography"
+import { Card, CardContent } from "@material-ui/core"
 
-import "./CompanyDetails.css"
+import EditIcon from '@material-ui/icons/Edit';
 
-const CompanyDetails = ({companyDetails, contactPersons,  events, orders}) => {
+import React from "react"
 
-    const {companyName, companyICO, billingAddress, contactAddress, created} = companyDetails;
+import "./css/CompanyDetails.css"
+
+
+const CompanyDetails = ({companyDetails, contactPersons, notes,  events, orders}) => {
+
+    const {companyName, companyICO, billingAddress, contactAddress, mainPhoneNumber, created, status, statusColor} = companyDetails;
```

Comment:
Neviem či by nebolo lepšie, aby si tieto informácie získal sám tento modul z backendu. Predalo by sa mu len ico a na základe toho by získal ostatné info.

Reply:
Tak ja som rátal, že to bude zisťovať nejaký nadradený komponent.

Reply:
Ja som to zas bral tak, že si každý komponent asynchrónne sám načítava dáta, ktoré potrebuje. Ale neviem ako to je lepšie, ja som to teda robil tak že tá tabuľka si sama načítava zoznam firiem, ale neviem či to má byť tak alebo takto, ako to robíš ty 😄